includes/class-wc-customer.php

```diff
@@ -242,6 +242,25 @@ public function has_calculated_shipping() {
 		return $this->get_calculated_shipping();
 	}
 
+	/**
+	 * Indicates if the customer has a non-empty shipping address.
+	 *
+	 * Note that this does not indicate if the customer's shipping address
+	 * is complete, only that one or more fields are populated.
+	 *
+	 * @return bool
```

Comment:
Maybe it would be good to add `@since 5.3.0` versioning.

Reply:
Added.